### ISSUE TITLE
Question: what's the best strategy to tweak nginx.conf

### ISSUE THREAD

**Issue by itzg** [smashwilson/lets-nginx#6]
I am wanting to add a `proxy_request_buffering off` to the config, but that wouldn't be appropriate as a pull request to your image. Any ideas how to apply a tweak like that?

**Comment by smashwilson** [comment]
One way you could do this is by making an image that's a child of this one, and overriding the entrypoint script that templates the config:

```
FROM smashwilson/nginx.conf

ADD better-entrypoint.sh /entrypoint.sh
```

**Comment by itzg** [comment]
Your answer made me think of a PR-worthy enhancement:

How about breaking out the nginx.conf template into a standalone file and then process it similar to now, but instead with [this bash templating technique](http://pempek.net/articles/2013/07/08/bash-sh-as-template-engine/)? Then a derived image could `ADD` to replace just the `nginx.conf` leaving the cert, cron, etc setup logic exactly from your base image's entry point script.

**Comment by smashwilson** [comment]
Hmm! Clever. Sure, that sounds great.

**Comment by itzg** [comment]
Cool. I'll give that a shot in the next couple of days and send you a PR if that goes well.

**Comment by itzg** [comment]
Addressed by #7
